test/over_react_redux/connect_test.dart

```diff
@@ -268,44 +268,44 @@ main() {
         jacket = mount(
           (ReduxProvider()..store = store1)(
             (ConnectedCounter()
-                ..ref = (ref){ counterRef = ref; }
+                ..ref = counterRef
                 // make `decrement` increment
                 ..decrement = () {store1.dispatch(IncrementAction());}
                 ..currentCount = 900
             )('test'),
           ),
         );
         // `button-decrement` will be incrementing now
-        var dispatchButton = getByTestId(jacket.getInstance(), 'button-decrement');
+        var dispatchButton = queryByTestId(jacket.mountNode, 'button-decrement');
 
-        expect(getDartComponent<CounterComponent>(counterRef).props.decrement, isA<Function>());
+        expect(counterRef.current.props.decrement, isA<Function>());
 
         // state.count is at 0
-        expect(getDartComponent<CounterComponent>(counterRef).props.currentCount, 900);
-        expect(jacket.getNode().innerHtml, contains('Count: 900'));
+        expect(counterRef.current.props.currentCount, 900);
+        expect(jacket.mountNode.innerHtml, contains('Count: 900'));
 
         // Click button mapped to trigger `propFromDispatch` prop.
         click(dispatchButton); // state.count is now equal to 1
 
         // wait for the next tick for the async dispatch to propagate
         await Future(() {});
 
-        expect(getDartComponent<CounterComponent>(counterRef).props.currentCount, 1);
-        expect(jacket.getNode().innerHtml, contains('Count: 1'));
+        expect(counterRef.current.props.currentCount, 1);
+        expect(jacket.mountNode.innerHtml, contains('Count: 1'));
       });
     });
 
     group('Options', () {
       group('areOwnPropsEqual', () {
         test('', () {
           ConnectedCounter = connect<CounterState, CounterProps>(
-            areOwnPropsEqual: expectAsync2((next, prev) {
```

Comment:
Why was this removed?

Reply:
honestly, because i added somewhere else and i forgot which one it was that i added and ended up removing them all lol ill add these back.